x/upgrade/keeper/keeper.go

```diff
@@ -34,7 +34,7 @@ type Keeper struct {
 	storeKey           storetypes.StoreKey             // key to access x/upgrade store
 	cdc                codec.BinaryCodec               // App-wide binary codec
 	upgradeHandlers    map[string]types.UpgradeHandler // map of plan name to upgrade handler
-	versionSetter      xp.ProtocolVersionSetter        // implements setting the protocol version field on BaseApp
+	VersionSetter      xp.ProtocolVersionSetter        // implements setting the protocol version field on BaseApp
```

Comment:
We can rename this field to `baseapp` tbh, wdyt. It's really a reference to baseapp, even though we only need one method from it.